package.json

```diff
@@ -921,6 +959,21 @@
           "command": "vscode-couchbase.refreshQueryHistory",
           "when": "view == query-history",
           "group": "navigation"
+        },
```

Comment:
commandPalette should not contain these commands

        {
          "command": "vscode-couchbase.iq.newChat",
          "when": "view == couchbase-iq && isIqNewChatButtonBVisible",
          "group": "navigation@1"
        },
        {
          "command": "vscode-couchbase.iq.logout",
          "when": "view == couchbase-iq && isIqLogoutButtonBVisible",
          "group": "navigation@3"
        },
        {
          "command": "vscode-couchbase.iq.showSettings",
          "when": "view == couchbase-iq",
          "group": "navigation@2"

Reply:
Updated